napari/_qt/qt_viewer.py

```diff
@@ -914,13 +914,8 @@ def _open_files_dialog(self, choose_plugin=False, stack=False):
         """Add files from the menubar."""
         filenames = self._open_file_dialog_uni(trans._('Select file(s)...'))
 
-        if (filenames != []) and (filenames is not None):
```

Comment:
AFAICT the output from `_open_file_dialog_uni` should be `list[str]` also if we just check if `filenames` is truthy I think it would satisfy both the above checks?

Reply:
yes